webpack.config.js

```diff
@@ -18,7 +18,10 @@ function generateConfig(name) {
       process: false
     },
     devtool: 'source-map',
-    mode: compress ? 'production' : 'development'
+    mode: compress ? 'production' : 'development',
+    externals: {
+      '@ohos.net.http': '@ohos.net.http'
```

Comment:
With the node adapter, they were excluding it by putting a replacement in the browser section at the `package.json`.

https://github.com/axios/axios/blob/ea0d9c69f78070dae841c66efd7263e246cab845/package.json#L72

A new line with `"./lib/adapters/ohos.js": "./lib/adapters/xhr.js"` will prevent the hole adapter and its dependencies to be ignored, not just the `@ohos.net.http` dependency. If this adapter is meant to be used in the web and node, i think the extra bytes in the cdn code is too much, maybe an external library would fit better.

Reply:
thanks, added it to browser, ohos.js is only used in HarmonyOS

Reply:
Np!

By doing that, should you still use the externals property?